supabase/lib/storage_client.py

```diff
@@ -14,8 +16,8 @@ class SupabaseStorageClient(StorageBucketAPI):
     >>> list_files = storage_file.list("something")
     """
 
-    def __init__(self, url, headers):
+    def __init__(self, url: str, headers: Dict[str, str]):
         super().__init__(url, headers)
 
-    def StorageFileAPI(self, id_):
+    def StorageFileAPI(self, id_: str):
```

Comment:
```suggestion
    def StorageFileAPI(self, id_: str) -> StorageFileAPI:
```
Missing return type annotation